pkg/services/textanalytics/plan_schemas.go

```diff
@@ -0,0 +1,29 @@
+package textanalytics
+
+import (
+	"github.com/Azure/open-service-broker-azure/pkg/azure"
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func generateProvisioningParamsSchema() service.InputParametersSchema {
+	return service.InputParametersSchema{
+		RequiredProperties: []string{"location", "resourceGroup"},
+		PropertySchemas: map[string]service.PropertySchema{
+			"location": &service.StringPropertySchema{
+				Title: "Location",
+				Description: "The Azure region in which to provision" +
+					" applicable resources.",
+				CustomPropertyValidator: azure.LocationValidator,
+			},
+			"resourceGroup": &service.StringPropertySchema{
+				Title: "Resource group",
+				Description: "The (new or existing) resource group with which" +
+					" to associate new resources.",
+			},
+			"name": &service.StringPropertySchema{
```

Comment:
Why make parameter `name` user-specified? In other modules, the name of the server, database, and so on, is generated randomly and is not configurable for users. I'm not familiar with textanalytics, please tell me if there is any special concern to do this. If not, could you make `name` randomly generated just like other modules?

Reply:
Thanks for catching this @norshtein . The parameter was actually unused and has been removed.